cmd/destroy.go

```diff
@@ -39,6 +41,17 @@ if the registry has already been deleted.`,
 		if err != nil {
 			log.Panic(err)
 		}
+		dryRun, err := cmd.Flags().GetBool("dry-run")
+		if err != nil {
+			log.Panic(err)
```

Comment:
it will avoid stack trace / and will return the error message to the user
```suggestion
			log.Println(err)
			return
```

Reply:
I am trying to following the existing pattern, not refactor anything at the moment.